package.json

```diff
@@ -53,6 +53,7 @@
     "react-redux": "^4.0.0",
     "react-router": "^1.0.0-rc3",
     "react-router-bootstrap": "^0.19.3",
+    "react-tabs": "^0.5.5",
```

Comment:
Why not [react-bootstrap's tabs](http://react-bootstrap.github.io/components.html#tabs)?